lib/Runtime/Library/MathLibrary.cpp

```diff
@@ -816,6 +816,23 @@ namespace Js
     {
         double result = 0;
 
+#ifdef POW_DOESNT_SUPPORT_NEGATIVE_INFINITY
+        if ( signbit( x ) && 0 == NumberUtilities::LuLoDbl( x ) && 0 == ( NumberUtilities::LuHiDbl( x ) & 0x7FFFFFFF ) )
```

Comment:
More curiosity than anything else- can we instead of this do *((int64*) &x) == Js::NumberConstants::k_NegZero? Either way, it would be nice to have this refactored out into its own function in NumberUtilities for readability.